addons/base_import/models/base_import.py

```diff
@@ -626,6 +637,21 @@ def _parse_import_data(self, data, import_fields, options):
                 # We should be able to manage both case
                 index = import_fields.index(name)
                 self._parse_float_from_data(data, index, name, options)
+            elif field['type'] == 'binary' and name in import_fields:
```

Comment:
This strikes me as problematic, though I don't have a solution: not all binary fields are base64-encoded images…

Maybe check for the `attachment` flag as well? Images should generally be stored as attachments.